CHANGELOG.md

```diff
@@ -2,3 +2,7 @@
 
 ## [0.1.0]
 - Initial implementation
+- Implement the basic encryption features for anonymous and shared-key encryption.
```

Comment:
anonymous and authenticated (or shared-secret) ? We renamed stuff to that from shared_key so we shouldn't be using that term

Reply:
Forgot to change this. Fixed now, thanks!